src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -107,11 +108,12 @@ const ReportActionItemFragment = (props) => {
             // we render it as text, not as html.
             // This is done to render emojis with line breaks between them as text.
             const differByLineBreaksOnly = Str.replaceAll(html, '<br />', '\n') === text;
+            const isWeb = [CONST.PLATFORM.WEB, CONST.PLATFORM.DESKTOP].includes(getPlatform());
```

Comment:
@pubudu-ranasinghe Why is this necessary? We discourage the use of platform dependent code unless it is absolutely necessary.

Reply:
This issue doesn't happen on native apps. And providing `inline-block` to RN styles apparently throws an error. So needed to contain it only to desktop and web.

Reply:
What is the error? If it is not supported by react-native, we better look for other solutions.